index.js

```diff
@@ -2780,7 +2770,14 @@
       } :
       wrapNext ({}, [], 0);
 
-    wrapped[inspect] = wrapped.toString = always0 (typeSignature (typeInfo));
+    wrapped.toString = always0 (typeSignature (typeInfo));
+    /* istanbul ignore else */
+    if (
+      typeof process !== 'undefined' &&
```

Comment:
@davidchambers This is a great addition to sanctuary-def - I will try it out on scrimba.com soon.

I only want to add that `process !== 'undefined'  && process != null` seems redundant. 

`process != null` should be enough.

```js
null == undefined // true
null != undefined // false
```

[The Abstract Equality Comparison Algorithm](https://262.ecma-international.org/5.1/#sec-11.9.3) says in step 2: _If x is null and y is undefined, return true._ Step 3 is similar but with `x` and `y` switched.

Reply:
ah, you don't want to create a global variable called `process` in non-node environments... Got it! Please disregard my comment ;)

Reply:
Also I think you pointed out in the chat that not guarding `process` will throw an error....

Reply:
The expression is verbose, but I don't think any part of it can safely be omitted.